src/designer/elsa-workflows-studio/src/components/screens/workflow-definition-editor/elsa-workflow-definition-editor-screen/elsa-workflow-definition-editor-screen.tsx

```diff
@@ -451,10 +569,153 @@ export class ElsaWorkflowDefinitionEditorScreen {
             {this.renderPublishButton()}
           </div>
         </div>
+        {this.renderTestActivityMenu()}  
       </div>
     );
   }
 
+  async onUseAsSchemaClick(message: WorkflowTestActivityMessage) {
+    const value = message.data["Body"];
+    let workflowModel = {...this.workflowModel};
+    const activityModel = workflowModel.activities.find(x => x.activityId == message.activityId);
+
+    const context: ActivityUpdatedContext = {
+      activityModel: activityModel,
+      data: JSON.stringify(convert(value), null, 1)
+    };
+    eventBus.emit(EventTypes.ActivityPluginUpdated, this, context);
+  }  
```

Comment:
The workflow editor shouldn't have any awareness of the Schema stuff. That's all specific to the HTTP endpoint activity.